drivers/flash/flash_npcx_fiu_nor.c

```diff
@@ -278,27 +284,40 @@ static int flash_npcx_nor_erase(const struct device *dev, off_t addr,
 		return -EINVAL;
 	}
 
+	if ((config->min_erase_size != BLOCK_4K_SIZE && config->min_erase_size != BLOCK_64K_SIZE) ||
+	    (config->max_erase_size != BLOCK_4K_SIZE && config->max_erase_size != BLOCK_64K_SIZE)) {
+		return -EINVAL;
```

Comment:
Since the type of `max-erase-size` and `max-erase-size` are `enum`, we might needn't this check.

Reply:
Yes, but I added it because if someone adds the new options in enum in node definition and don't support that erase block in code, then the `while` will go through infinity loop. hence added a check.

Reply:
Removed the checks as they are no longer valid, thank you